src/snowcli/cli/stage/commands.py

```diff
@@ -71,9 +71,9 @@ def stage_put(
         writable=True,
         resolve_path=True,
         help=(
-            "File or directory to upload to stage, can include a * in the path, "
-            'like "folder/*.csv". Make sure you put quotes around the path if it'
-            " includes a *. "
+            "File or directory to upload to stage, can include a \* in the path, "
+            "like ``folder/*.csv``\. Make sure you put quotes around the path if it"
+            " includes a \*. "
```

Comment:
During docs build this line caused errors, had to change it

Reply:
Let's be consistent with using `. AFAIK the double ` will render as code sample in rst.

Reply:
Checked one more option and it could be ``File or directory to upload to stage, can include a `*` in the path, like `folder/*.csv`. Make sure you put quotes around the path if it includes a `*`.``

Reply:
This option looks good.